app/uk/gov/hmrc/exports/models/declaration/ExportsDeclaration.scala

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmrc.exports.models.declaration
+
+import java.time.{Instant, LocalDateTime, ZoneOffset}
+import java.util.UUID
+
+import play.api.libs.json.{JsError, JsNumber, JsObject, JsResult, JsSuccess, JsValue, Json, OFormat}
+
+case class ExportsDeclaration
+(
+  id: String = UUID.randomUUID().toString,
+  eori: String
+)
+
+object ExportsDeclaration {
+  implicit val formatInstant: OFormat[LocalDateTime] = new OFormat[LocalDateTime] {
```

Comment:
This is required to save dates in the format Mongo wants rather than as a String

Reply:
In hindsight It should be an Instant as agreed in the last tech meeting. Will change in subsequent PRs

Reply:
so we're storing dates in mongo in EpochMilli format?